weatherflow2mqtt/helpers.py

```diff
@@ -913,3 +913,78 @@ def zambretti_forecast(self, z_num: int):
         z_text += z_forecast[int(z_num)]
 
         return self.translations["zambretti"][z_text]
+
+    def fog_probability(self, solar_elevation, wind_speed, humidity, dew_point):
+        """ Return probability of fog in percent.
+        Input:
+            Solar Elevation - determine daylight
+            Wind Speed
+            Humidity
+            Dew Point
+        Where:
+        """
+        if (
+            solar_elevation is None
+            or wind_speed is None
+            or humidity is None
+            or dew_point is None
+        ):
+            return None
+        
+        fog = 0
+        diff = temperature - dew_point
```

Comment:
Where is this temperature value coming from?

Reply:
I decided that this was the current air temperature, and supplied that to the formula. If this is not correct, let me know.